flocker_bb/builders/maint.py

```diff
@@ -25,6 +43,128 @@ def makeCleanOldBuildsFactory():
     return factory
 
 
+def get_creation_time(node):
+    """
+    Get the creation time of a libcloud node.
+
+    Rackspace and EC2 store the information in different metadta.
+
+    :return datetime:
+    """
+    date_string = node.extra.get("created", node.extra.get("launch_time"))
+    if date_string is None:
+        return None
+    else:
+        return parse_date(date_string)
+
+
+@attributes([
+    Attribute('lag'),
+    Attribute('prefixes',
+              default_value=('acceptance-test-', 'client-test-')),
```

Comment:
:question: I've often thought it would be nice to be able to configure this prefix so that acceptance nodes launched by the "RichardW" staging master  are easily distinguishable from those created by other masters and can be easily identified and cleaned up manually if necessary.